ui/raidboss/data/05-shb/trial/wol-ex.js

```diff
@@ -0,0 +1,443 @@
+          en: 'Protean',
+          de: 'Himmelsrichtungen',
+          fr: 'Position',
+          ja: '散開',
+          cn: '散开',
+          ko: '위치 산개',
+        }[data.displayLang];
+        data.imbued.push(data.imbuedStone);
+      },
+    },
+    {
+      id: 'WOLEx Imbued Coruscance In',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F4A', capture: false }),
+      preRun: function(data) {
+        data.imbued = data.imbued || [];
+        data.imbuedSwordIn = {
+          en: 'In',
+          de: 'Rein',
+          fr: 'Intérieur',
+          ja: '中へ',
+          cn: '靠近',
+          ko: '안으로',
+        }[data.displayLang];
+        data.imbued.push(data.imbuedSwordIn);
+      },
+      alertText: function(data) {
+        let msg = data.imbued.join(' + ');
+        delete data.imbued;
+        return msg;
+      },
+    },
+    {
+      id: 'WOLEx Imbued Coruscance Out',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F49', capture: false }),
+      preRun: function(data) {
+        data.imbued = data.imbued || [];
+        data.imbuedSwordOut = {
+          en: 'Out',
+          de: 'Raus',
+          fr: 'Exterieur',
+          ja: '外へ',
+          cn: '远离',
+          ko: '밖으로',
+        }[data.displayLang];
+        data.imbued.push(data.imbuedSwordOut);
+      },
+      alertText: function(data) {
+        let msg = data.imbued.join(' + ');
+        delete data.imbued;
+        return msg;
+      },
+    },
+    {
+      id: 'WOLEx The Bitter End',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F0A' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBusterSwap(),
+    },
+    {
+      id: 'WOLEx Summon Wyrm',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F41', capture: false }),
+      delaySeconds: 6,
+      infoText: {
+        en: 'Avoid Wyrm Dash',
+      },
+    },
+    {
+      id: 'WOLEx Absolute Flash',
+      netRegex: NetRegexes.headMarker({ id: '00B3' }),
+      suppressSeconds: 5,
+      response: Responses.lookAwayFromTarget(),
+    },
+    {
+      id: 'WOLEx Elddragon Dive',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F0B', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'WOLEx Fatal Cleave / Blade Of Shadow',
+      // Either tank buster, but don't be too noisy
+      netRegex: NetRegexes.startsUsing({ source: ['Spectral Warrior', 'Spectral Dark Knight'], id: '515[47]', capture: false }),
+      condition: Conditions.caresAboutPhysical(),
+      suppressSeconds: 2,
+      infoText: {
+        en: 'Tank Busters',
+        de: 'Tankbuster',
+        fr: 'Tank buster',
+        ja: 'タンクバスター',
+        cn: '坦克死刑',
+        ko: '탱버',
+      },
+    },
+    {
+      id: 'WOLEx Berserk / Deep Darkside',
+      // Either silence, but don't be too noisy
```

Comment:
I would maybe argue that interrupts that wipe the raid should get an alarm, and it's worth being noisy here?

Reply:
Noisy was relative to "spamming this twice," so I am onboard with bumping it to alarm.